Philips.CodeAnalysis.SecurityAnalyzers/AvoidPasswordAnalyzer.cs

```diff
@@ -19,6 +19,10 @@ public class AvoidPasswordAnalyzer : SingleDiagnosticAnalyzer
 		public const string MessageFormat = @"Naming something Password suggests a potential hard-coded password.";
 		private const string Description = @"Avoid hard-coded passwords.  (Avoid this analyzer by not naming something Password.)";
 
+		private const string MsTestMetadataReference = "Microsoft.VisualStudio.TestTools.UnitTesting.Assert";
+
+		public virtual bool ShouldAnalyzeTests { get; set; }

```

Comment:
The default now is false, which effectively turns it off, or Am I missing something?

Reply:
I disabled the analyzer when in Unit Tests. This flag allows for forcing it back on in Unit Tests. (It will always run if no Unit Test Reference.)